config/vufind/obalkyknih.ini

```diff
@@ -0,0 +1,36 @@
+; Configuration of ObalkyKnih.cz content service
+; It does support this types of content:
+;   - Book covers
+;   - Periodics covers (general and individual issues)
+;   - Table of contents (OCR text and image preview)
+;   - User comments
+;   - User ratings (aggregated from more 3rd party services and its own)
+;   - Summaries
+;   - Authorities pictures
+;   - Citations (ISO 690)
+;   - Links to free e-books
+;   - Links to digitalized books in Czech libraries
+;   - Document recommendations based on libraries checkouts data
+; Please note, that not all service listed above are implemented in VuFind
+;
+; Content could be found by these identifiers: ISBN, ISSN, ISMN, OCoLC, EAN (UPC),
+; NBN (number of Czech national bibliography) and own identifier (identifier from a local library system)
+;
+; Note: You have to register an account on obalkyknih.cz and configure your VuFind's instance IP address/referrer url
+
+[ObalkyKnih]
+base_url1 = https://cache.obalkyknih.cz
```

Comment:
What is the purpose of `base_url1`/`base_url2`? Would it be better to implement this as `base_url[]` so an arbitrary number could be configured as needed?

Reply:
ObalkyKnih service has two servers, when one is down, api client could use the second one as fallback (not implemented in my code yet).

But you are right, that array would be better here, I'll fix it.

Reply:
Done just now